Dart/src/com/jetbrains/lang/dart/highlight/DartColorsAndFontsPage.java

```diff
@@ -179,8 +218,7 @@ public String getDemoText() {
            "var <top.level.var.decl>topLevelVar</top.level.var.decl> = new SomeClass(null).<inherited.call>g</inherited.call>();\n\n" +
            "<top.level.func.decl>main</top.level.func.decl>() {\n" +
            "  <top.level.func.call>print</top.level.func.call>(<top.level.var.call>topLevelVar</top.level.var.call>);\n" +
-           "}\n"
-      ;
+           "}\n";
```

Comment:
Good time to update the whole text!
Now it is way too long, has incorrectly colored regions and at the same time doesn't demonstrate all options.